tests/results/SourceRangeTest.py

```diff
@@ -42,6 +44,15 @@ def test_from_clang_range(self):
         compare = SourceRange.from_values('t.c', 1, 2, 3, 4)
         self.assertEqual(uut, compare)
 
+    @patch('coalib.results.SourceRange.get_distribution')
+    def test_clang_version_check(self, get_distribution):
+        import importlib
+        import coalib.results.SourceRange
+        with self.assertRaisesRegexp(LibclangError,
+                                     'coala requires clang 3.4.0'):
+            get_distribution.configure_mode(version='1.2')
```

Comment:
if this line is setting the libclang version to 1.2 shouldn't the assert statement be after this to actually raise the error?

Reply:
can you explain why are you trying to reload the module `coalib.results.Diff`